test/sync_test.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
this script has to be integrated with the deploy.sh, just add additional parameter support.